.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node
+dist: trusty
+language: node_js
+node_js:
+  - "6.9.5"
+before_script:
+  - npm install
```

Comment:
Referred [this](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/) what is the problem with this?